index.bs

```diff
@@ -988,6 +1001,13 @@ spec:css-syntax-3;
     1.  If |interfaces| is [=set/empty=], then return[=a promise rejected with=]
         a "{{NotSupportedError}}" {{DOMException}}.
 
+    1.  [=set/For each=] |interface1| of |interfaces| and |interface2| of |interfaces|:
+
+        1. Let |type1| be |interface1|'s {{Credential/[[type]]}} and Let |type2| be |interface2|'s {{Credential/[[type]]}} 
+
+        1. If |type1|'s list of [=credential type registry/Types allowed in the same get() request=] doesn't contain |type2|, 
```

Comment:
We also need to skip types that are equal.

Reply:
Done!